xtask/Cargo.toml

```diff
@@ -13,7 +13,7 @@ risc0-fault = { path = "../risc0/fault" }
 risc0-zkp = { workspace = true }
 risc0-zkvm = { workspace = true, features = ["prove"] }
 risc0-zkvm-methods = { path = "../risc0/zkvm/methods" }
-tempfile = "3.5"
+tempfile = "3.3"
```

Comment:
I'm surprised by this version downgrade -- if there's a reason I'm not concerned, but I'm calling attention to it in case it was an accident.

Reply:
After removing the example from the Relay README, this change is not required anymore

Reply:
Nevermind:
```
cargo run --bin cargo-risczero --no-default-features -- risczero install
  shell: /usr/bin/bash -e {0}
  env:
    CARGO_TERM_COLOR: always
    MSYS: winsymlinks:nativestrict
    GITHUB_TOKEN: ***
    Updating git repository `[https://github.com/paritytech/polkadot-sdk/`](https://github.com/paritytech/polkadot-sdk/%60)
    Updating crates.io index
    Updating git repository `[https://github.com/w3f/ring-vrf`](https://github.com/w3f/ring-vrf%60)
    Updating git repository `[https://github.com/w3f/fflonk`](https://github.com/w3f/fflonk%60)
    Updating git repository `[https://github.com/w3f/ring-proof`](https://github.com/w3f/ring-proof%60)
error: failed to select a version for `tempfile`.
    ... required by package `xtask v0.1.0 (/opt/actions-runner/_work/risc0/risc0/xtask)`
versions that meet the requirements `^3.5` are: 3.8.1, 3.8.0, 3.7.1, 3.7.0, 3.6.0, 3.5.0

all possible versions conflict with previously selected packages.

  previously selected package `tempfile v3.3.0`
    ... which satisfies dependency `tempfile = "~3.3"` of package `cargo-generate v0.18.0`
    ... which satisfies dependency `cargo-generate = "^0.18"` of package `cargo-risczero v0.19.0 (/opt/actions-runner/_work/risc0/risc0/risc0/cargo-risczero)`

failed to select a version for `tempfile` which could resolve this conflict
```